Proiect_PAO/src/aplicatieBancara/Client/Client.java

```diff
@@ -0,0 +1,119 @@
+package aplicatieBancara.Client;
+
+import aplicatieBancara.Card.Card;
+import aplicatieBancara.Cont.Cont;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class Client {
+
+    private static int id = 0;
+    private final int idClient;
+    private String nume;
+    private String prenume;
+    private String CNP;
+    private Date dataNastere;
+    private String email;
+    private String telefon;
+    private Adresa adresa;
+    private ArrayList<Cont> conturi = new ArrayList<Cont>();
```

Comment:
de recomandat la instantiere obiect sa omiti in partea dreapta tipul 'Cont', diamond operator stie sa infereze din partea stanga: 

private ArrayList<Cont> conturi = new ArrayList<>();